docs/DeveloperGuide.md

```diff
@@ -850,6 +850,144 @@ testers are expected to do more *exploratory* testing. Alternative test cases wi
 - `remark 100000 r/Birthday on 31 July.`.
 - `remark 1 r/Birthday on 31 July r/`.
 
+### Schedule a meeting with a person
+
+#### Successful scheduling
+
+**Prerequisites:**
+1. List all persons using the `list` command. At least 5 persons are in the list.
+2. Assume the time for now is 14-04-2024T12:00.
+3. Assume no scheduled meeting with any person currently.
+
+**Example Test Case**: `schedule 1 s/23-04-2024T16:00 e/23-04-2024T17:00`.
+
+**Expected Result:**
+1. A meeting from 23-02-2024 16:00 to 23-02-2024 17:00 is attached to the person at index 1.
```

Comment:
For the datetime values lets keep it as is since the successful schedule output is as shown:
![image](https://github.com/AY2324S2-CS2103-F15-1/tp/assets/122505234/e6adafd5-657a-414e-8ef0-e92baa03a9f9)

Reply:
Yep this one I will change. Thanks!

Reply:
My main concern right now is due to using assumption, which I feel we should not be doing? If we are not going by that specific assumption the datetime may change accordingly.